cpp/src/io/orc/reader_impl.cu

```diff
@@ -1420,6 +1442,10 @@ table_with_metadata reader::impl::read(size_type skip_rows,
     create_columns(std::move(out_buffers), out_columns, schema_info, stream);
   }
 
+  if (not out_columns.empty()) {
+    printf("RGSL: null count is %d \n", out_columns[0]->null_count());
+    auto col = out_columns[0]->view();
+  }
```

Comment:
```suggestion
```

Reply:
These prints were meant for testing some failure in CI